### PR TITLE
refactor: use ref instead of pointer

### PR DESCRIPTION
**Pull request by WalterBright**
Redo inlineScan() to use ref for in/out references rather than pointers. Trivial.

### CODE REVIEW

File: src/inline.d [hunk 1]
```diff
@@ -1576,26 +1582,32 @@ public:
...
+        inlineScan(e.e1);
+        inlineScan(e.e2);
     }
 
-    void inlineScan(Expression* e)
+    /********************************
+     * Scan Statement s for inlining opportunities,
+     * and if found replace s with an inlined one.
+     * Params:
+     *  s = Statement to be scanned and updated
```
Comment (yebblies) [inline src/inline.d]: `s` doesn't match the parameter name.

Comment (yebblies) [inline src/inline.d]: And I guess this should say `Expression` everywhere?

Comment (WalterBright) [inline src/inline.d]: done

Comment (yebblies) [inline src/inline.d]: Yeah, I don't know how I managed to comment on the wrong one.

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**